nav2_collision_monitor/src/polygon.cpp

```diff
@@ -264,8 +264,21 @@ bool Polygon::getCommonParameters(std::string & polygon_pub_topic)
     }
 
     nav2_util::declare_parameter_if_not_declared(
-      node, polygon_name_ + ".max_points", rclcpp::ParameterValue(3));
-    max_points_ = node->get_parameter(polygon_name_ + ".max_points").as_int();
+      node, polygon_name_ + ".min_points", rclcpp::ParameterValue(4));
```

Comment:
Why change the defaults?

Reply:
This appears since `max_points` parameter meant the maximum number of points where we are still not triggering the action, while `min_points` - is a number starting from the action to be initiated. In other words, `min_points = max_points + 1`.
Here we are:
```
-    if (getPointsInside(points_transformed) > max_points_) {
+    if (getPointsInside(points_transformed) >= min_points_) {
```